lib/classes/CLI.js

```diff
@@ -149,6 +149,13 @@ class CLI {
     const command = this.serverless.pluginManager.getCommand(commandsArray);
     const commandName = commandsArray.join(' ');
 
+    // Remove entrypoint subcommands
+    _.forEach(command.commands, (details, cmd) => {
```

Comment:
This will only remove top-level entry points. As commands can contain multiple levels of subcommands, this won't work in case a sub-sub command would be an entrypoint.

A solution would be to fetch all commands and then just render the command out of that.
```
const allCommands = this.serverless.pluginManager.getCommands();
const command = _.reduce(commandsArray, (currentCmd, command) => {
  return currentCmd[command];
}, commandsArray);
// Now command contains the command given by commandsArray,
// or undefined, if any parent of the selected command is an entrypoint
// or the command does not exist
if (command) {
  // Command found
} else {
  // Command not found
}
```